lib/locales/it.js

```diff
@@ -14,15 +14,15 @@ module.exports = {
     dictionary: {
         'Autoscale': 'Scala automaticamente',                                                               // components/modebar/buttons.js:139
         'Box Select': 'Selezione box',                                                                    // components/modebar/buttons.js:103
-        'Click to enter Colorscale title': 'Clicca per inserire un titolo alla scala di colori',                     // plots/plots.js:437
+        'Click to enter Colorscale title': 'Clicca per inserire un titolo alla scala colori',                     // plots/plots.js:437
         'Click to enter Component A title': 'Clicca per inserire un titolo al componente A',            // plots/ternary/ternary.js:386
         'Click to enter Component B title': 'Clicca per inserire un titolo al componente B',            // plots/ternary/ternary.js:400
         'Click to enter Component C title': 'Clicca per inserire un titolo al componente C',            // plots/ternary/ternary.js:411
         'Click to enter Plot title': 'Clicca per inserire un titolo al grafico',                        // plot_api/plot_api.js:579
         'Click to enter X axis title': 'Clicca per inserire un titolo all\'asse X',                      // plots/plots.js:435
         'Click to enter Y axis title': 'Clicca per inserire un titolo all\'asse Y',                      // plots/plots.js:436
         'Click to enter radial axis title': 'Clicca per inserire un titolo per l\' asse radiale',                      // plots/plots.js:436
-        'Compare data on hover': 'Paragona i dati al passaggio del mouse',                            // components/modebar/buttons.js:167
+        'Compare data on hover': 'Compara i dati al passaggio del mouse',                            // components/modebar/buttons.js:167
         'Double-click on legend to isolate one trace': 'Doppio click per isolare i dati di una traccia',  // components/legend/handle_click.js:90
         'Double-click to zoom back out': 'Doppio click per tornare allo zoom iniziale',                                  // plots/cartesian/dragbox.js:299
```

Comment:
The community member suggests "Doppio click per ridurre" - please let us know what you think.

Reply:
"Doppio click per ridurre" is ok. I'd say "Doppio click per ridurre lo zoom" as less ambiguous